contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```
@@ -1330,33 +1329,17 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
         depositableValidatorsCount = totalMaxValidators - totalDepositedValidators;
     }
 
-    function _isOperatorPenalized(Packed64x4.Packed memory stuckPenaltyStats) internal view returns (bool) {
-        return stuckPenaltyStats.get(REFUNDED_VALIDATORS_COUNT_OFFSET) < stuckPenaltyStats.get(STUCK_VALIDATORS_COUNT_OFFSET)
-            || block.timestamp <= stuckPenaltyStats.get(STUCK_PENALTY_END_TIMESTAMP_OFFSET);
+    /// @notice [DEPRECATED] Penalty logic removed. Always returns `false`.
+    /// @dev _nodeOperatorId Ignored.
+    /// @return Always returns `false`.
+    function isOperatorPenalized(uint256 /* _nodeOperatorId */) public pure returns (bool) {
```

The same suggestion as in [this comment](https://github.com/lidofinance/core/pull/1018/files#r2107003367).

If we keep these functions, it might be confusing. For example, see the integration tests in
`protocol-happy-path.integration.ts`. I suggest deleting the function so that any places where the flow might change will fail explicitly, making it easier to apply the appropriate fix.